src/server/pachyderm_test.go

```diff
@@ -1102,26 +1121,47 @@ func TestPipelineState(t *testing.T) {
 		"",
 		false,
 	))
+
 	// Wait for pipeline to get picked up
 	time.Sleep(15 * time.Second)
+	require.NoError(t, backoff.Retry(func() error {
+		pipelineInfo, err := c.InspectPipeline(pipeline)
+		if err != nil {
+			return err
+		}
+		if pipelineInfo.State != pps.PipelineState_PIPELINE_RUNNING {
+			return fmt.Errorf("no running pipeline")
+		}
+		return nil
+	}, backoff.NewExponentialBackOff()))
```

Comment:
This will run for 15 minutes if it doesn't find the pipeline state which is probably longer than is ever going to be needed. Why don't we set it to something like a minute?